apps/docs-app/docs/experiments/sfc/index.md

```diff
@@ -0,0 +1,344 @@
+            done: !item.done,
+          };
+        }
+        return item;
+      })
+    );
+  }
+</script>
+
+<template>
+  <div class="container">
+    <h1>Todo list</h1>
+    <ul class="todoList">
+      @for (item of list(); track $index) {
+      <li class="todoItem">
+        <span>{{ item.name }}</span>
+        <input
+          type="checkbox"
+          [checked]="item.done"
+          (change)="onDone(item.id)"
+        />
+      </li>
+      }
+    </ul>
+    <form (submit)="onSubmit($event)">
+      <h2>Add a task</h2>
+      <label>
+        <div>Task name</div>
+        <input name="name" />
+      </label>
+      <button>Add task</button>
+    </form>
+  </div>
+</template>
+
+<style>
+  .container {
+    max-width: 600px;
+    margin: 0 auto;
+    padding: 1rem;
+    text-align: center;
+  }
+
+  .todoList:empty::after {
+    content: 'No tasks';
+  }
+
+  .todoItem {
+    display: flex;
+    justify-content: space-between;
+  }
+</style>
+```
+
+# Usage
+
+To use the Analog SFC format, you'll need to use the Analog Vite plugin or the [Analog Astro plugin](/docs/packages/astro-angular/overview) with an additional flag to enable its usage:
+
+```typescript
+import { defineConfig } from 'vite';
+import analog from '@analogjs/vite-plugin-angular';
+
+export default defineConfig({
+  // ...
+  plugins: [
+    analog({
+      experimental: {
+        // Required to use the Analog SFC format
+        supportAnalogFormat: true,
+      },
+    }),
+  ],
+});
+```
+
+# Metadata
+
+While class decorators are used to add metadata to a component or directive in the traditional Angular authoring methods, they're replaced in the Analog format with the `defineMetadata` global function:
+
+```typescript
+defineMetadata({
+  host: { class: 'block articles-toggle' },
+});
+```
+
+This supports all of the decorator properties of `@Component` or `@Directive` with a few exceptions.
+
+## Disallowed Metadata Properties
+
+The following properties are not allowed on the metadata fields:
+
+- `template`: Use the SFC `<template>` or `defineMetadata.templateUrl` instead
+- `standalone`: Always set to `true`
+- `changeDetection`: Always set to `OnPush`
+- `styles`: Use the SFC `<style>` tag
+- `outputs`: Implicitly added with `new EventEmitter()` usage
+- `inputs`: Use the `input` signal API instead
+
+# Using Components
+
+When using the Analog format, you do not need to explicitly export anything; the component is the default export of the `.analog` file:
+
+```typescript
+import { bootstrapApplication } from '@angular/platform-browser';
+import App from './app/app.analog';
+import { appConfig } from './app/app.config';
+
+bootstrapApplication(App, appConfig).catch((err) => console.error(err));
+```
+
+Further, when importing an `.analog` file into another `.analog` file, you do not need to add them to your `imports` like you do with non-analog components and directives.
+
+Instead, you can utilize components directly from their imported name:
```

Comment:
The selector is not determined by the import name, more context here: https://github.com/analogjs/analog/pull/912

Note that although it is possible to import an analog component without an import name, I believe both Brandon/Chau prefer the named import as the default

Personally, I think the solution here is just a bit awkward either way until Angular implements their double-import solution at which point that would just be adopted in Analog